doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -185,14 +170,121 @@ Traceability to Reference Model.
 | `req.sec.ntw.05` | Networking | The Architecture **must** have the underlay network include strong access controls that adhere to the V1.1 NIST Cybersecurity Framework. | |
+<!-- | `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. | |
+| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with applicable security standard (national, regional), for example ISO27001. | | -->
+
 <p align="center"><b>Table 2-8:</b> OpenStack Security Requirements.</p>
 
+
+
+<a name="2.4"></a>
+### 2.4 Architecture and OpenStack Recommendations
+
+The requirements listed in this section are optional, viz. do not need to be implemented in conformant implementations.
+
+<a name="2.4.1"></a>
+2.4.1 General Recommendations
+
+| Ref # | sub-category | Description |  Traceability |
+|----|-------|---------------------|-------------|
+| `rec.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. | OpenStack consists of both stateless and stateful services where the stateful services utilize a database. For latter see "[Configuring the stateful services](https://docs.openstack.org/ha-guide/control-plane-stateful.html)"|
+| `rec.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. | |
+| `rec.gen.scl.01` | Scalability | The Architecture **should** support policy driven auto-scaling. | This requirement is currently not addressed but will likley be supported through [Senlin](https://docs.openstack.org/senlin/pike/index.html), cluster management service. |
+| `rec.gen.rsl.02` | Resiliency | The Architecture **should** support resilient OpenStack service components that are not subject to `req.gen.rsl.01`. | |
+
+<p align="center"><b>Table 2-9:</b> General Recommendations</p>
+
+
+<a name="2.4.2"></a>
+2.4.2 Infrastructure Recommendations
+
+| Ref # | sub-category | Description |  Traceability |
+|----|-------|---------------------|-------------|
+| `rec.inf.com.02` | Compute | The Architecture **should** include industry standard hardware management systems at both HW device level (embedded) and HW platform level (external to device). | |
+| `rec.inf.com.03` | Compute | The Architecture **should** support symmetrical CPU multi-processing with shared memory access as well as multi-threading. | |
+| `rec.inf.stg.05` | Storage | The Architecture **should** be able to accommodate VNFs that store back into its image through use of hypervisor attached volumes. | **Wouldn't that be a Security Violation** |
```

Comment:
Can we remove this requirement (rec.inf.stg.05)? It is not only a security problem. It is against NFV architecture to allow a VNF write into the package repository and btw. the packages are encrypted. Also, how to deal with multiple instances of such VNF?
Why would anybody need that?

Reply:
@ulikleber Glad to do it. Thanks